gen/internal/tests/thrift/nozap.thrift

```diff
@@ -0,0 +1,17 @@
+enum EnumDefault {
+    Foo, Bar, Baz
+}
+
+struct PrimitiveRequiredStruct {
+    1: required bool boolField
+    2: required byte byteField
+    3: required i16 int16Field
+    4: required i32 int32Field
+    5: required i64 int64Field
+    6: required double doubleField
+    7: required string stringField
+    8: required binary binaryField
+}
+
+typedef i64 Timestamp  // alias of primitive
+typedef string State
```

Comment:
These typedef cases won't get MarshalLogObject/Array even with the flag.
 Can we replace them with,

    typedef map<string, string> StringMap
    typedef PrimitiveRequiredStruct Primitives
    typedef list<string> StringList

This will exercise the cases we care about, I think.